crates/wallet/src/wallet/mod.rs

```diff
@@ -1763,16 +1763,15 @@ impl Wallet {
         )
     }
 
-    /// Return the "public" version of the wallet's descriptor, meaning a new descriptor that has
+    /// Returns the descriptor used to create addresses for a particular `keychain`.
+    /// It's the "public" version of the wallet's descriptor, meaning a new descriptor that has
     /// the same structure but with every secret key removed
     ///
     /// This can be used to build a watch-only version of a wallet
```

Comment:
⛏️ Docs should be proper sentences ending with a period and now is a good time to clarify that public keys correspond to the descriptors secret keys. 
```suggestion
    /// the same structure but with the all secret keys replaced by their corresponding public key.
    ///
    /// This can be used to build a watch-only version of a wallet.
```

Reply:
Thanks for the suggestion, just commited.